onnxruntime/core/providers/tvm/tvm_ep_options.h

```diff
@@ -22,12 +24,14 @@ const std::string LLVM_TARGET_AVX2 = "llvm -mcpu=core-avx2";
 const std::string LLVM_TARGET_SKYLAKE_AVX512 = "llvm -mcpu=skylake-avx512";
 const std::string LLVM_TARGET_AVX512 = "llvm -mcpu=skylake-avx512";
 }  // namespace cpu_targets
-}  // namespace tvm
 
-using TVMInputShapes = std::unordered_map<std::string, std::vector<int64_t>>;
+using TVMTensorShape = std::vector<int64_t>;
```

Comment:
Maybe that would be worth replacing it by TensorShapeVector? This container uses the stack for small shapes.

Reply:
Thanks @xadupre for useful tip! I have replaced TVMTensorShape by TensorShapeVector and fixed some places related it for correct work. The Pr can be rechecked by CI again